extensions/amp-auto-lightbox/0.1/amp-auto-lightbox.js

```diff
@@ -386,6 +386,10 @@ function generateLightboxUid() {
  */
 export function apply(ampdoc, element) {
   return Mutation.mutate(element, () => {
+    // amp-lightbox-gallery will reject elements with a set `on` attribute.
+    // Since the attribute is resolved to be valid before applying, this is ok
+    // to do.
+    element.removeAttribute('on');
```

Comment:
mm, this is probably harmless in all cases but not very comfortable removing `on` from user-code, even if invalid. also this removes `on` always but check is for `tap` only ( although I don't think there is any other events beside tap for images anyway ).

Anyway, let's make `meetsHeuristicsForTap_` in lightbox to use the new `hasResolvableAction`?